maintainers/maintainer-list.nix

```diff
@@ -6900,6 +6900,12 @@
       fingerprint = "5CB5 9AA0 D180 1997 2FB3  E0EC 943A 1DE9 372E BE4E";
     }];
   };
+  iogamaster = {
```

Comment:
this can probably be removed now, since it's not used anywhere

though if you do want to maintain something, feel free to do it in a separate PR :)